### ISSUE TITLE
Учимся создавать pull request :)

### ISSUE THREAD

**Issue by PanovaElena** [OLees/Polinom#2]
@OLees, это важно, потому что у меня появляется возможность комментировать отдельные строчки кода. Итак:

1) Создаем репозиторий, указываем "добавить README", чтобы автоматически создался первый коммит.
2) Клонируем репозиторий.
3) Создаем пользовательскую ветку, переходим в нее (команда git checkout).
4) Пишем код, делаем коммиты в эту ветку.
5) Заливаем код на гитхаб.
6) Делаем pull request из вашей ветки в main/master/..., не мержим его и не закрываем, а в комментариях указываем @PanovaElena 

В общем, ничего сложного, нужно только быть внимательным.

По коду: все нормально, только обычно функция получения указателя в итераторе называется просто get(), можно назвать getPtr() или как-нибудь еще, но getit() сбивает с толку.

Принято.

**Comment by OLees** [comment]
Да, я понял. Просто я заливал вечером, забыв изначально создать проект, поэтому пришлось создавать так. Репозиторий по таблицам уже создал)
 
 
С уважением,
Даниил Никитин
***@***.***
